src/output-json.c

```diff
@@ -348,6 +359,11 @@ static void EveAddFlowVars(const Flow *f, SCJsonBuilder *js_root, SCJsonBuilder
         SCJbSetObject(js_root, "flowints", js_flowints);
         SCJbFree(js_flowints);
     }
+    if (js_flowfloats) {
+        SCJbClose(js_flowfloats);
+        SCJbSetObject(js_root, "flowfloats", js_flowfloats);
```

Comment:
To me, this is good implementation but adding output to a special variable is a bit confusing as the others in such league are keywords themselves. `flowvar`, `flowbit`, `flowint` etc

Reply:
@showipintbri -- this output seems ok but I would like suggestions.

Here's an `entropy` value within a flow record
```
{
  "timestamp": "2009-10-02T12:02:44.364652-0400",
  "flow_id": 598561275346839,
  "event_type": "flow",
  "src_ip": "192.168.2.8",
  "src_port": 1099,
  "dest_ip": "209.34.91.23",
  "dest_port": 80,
  <trimmed> 
  "metadata": {
    "flowints": {
      "tcp.retransmission.count": 1
    },
    "flowfloats": {
      "entropy": 5.171311464031578
    }
  },
  <trimmed>
  ```

Reply:
```json
{
    "metadata": {
        "entropy": [
             { "file.data": 7.1 },
             { "http.uri": 1.1 },
        ]
    }
}
```